scripts/Gossip.js

```diff
@@ -40,28 +40,57 @@ var Gossip = function(robot) {
     });
   }
 
-  var findGossip = function(text, msg) {
+  var findGossip = function(text, msg, max_rows = 0) {
```

Comment:
Can you take the `gossip haiku` code and put it into a different branch so it doesn't clobber this PR?